ibis/expr/operations.py

```diff
@@ -1742,10 +1742,9 @@ def _validate_args(self, args: List[str]):
             setattr(self, arg, value)
 
 
-class Union(TableNode, HasSchema):
+class SetOperation(TableNode, HasSchema):
     left = Arg(rlz.noop)
     right = Arg(rlz.noop)
-    distinct = Arg(rlz.validator(bool), default=False)
 
     def _validate(self):
         if not self.left.schema().equals(self.right.schema()):
```

Comment:
I think the error message is not correct here (it still says union in it), can you please fix?

Reply:
Maybe just say "Table schemas must be equal for set operations"

Reply:
Yeah sounds good